packages/demos-babylon/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'eval-source-map',
+  resolve: {
+    modules: ['node_modules'],
+  },
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        {
+          from: '**/*.*',
+          context: 'src/',
```

Comment:
Is this equal to `{from: 'src/**/*.*'}`?

Reply:
No. If use the pattern you specify it will create a "src" subdirectory under the "dist" directory and copy files into that.